parmed/amber/_amberparm.py

```diff
@@ -379,7 +380,8 @@ def from_structure(cls, struct, copy=False):
                 dt.per = 1.0
         inst.remake_parm()
         inst._set_nonbonded_tables(nbfixes)
-        if inst.pointers['NCOPY'] >= 2:
+        n_copy = inst.pointers.get('NCOPY', 1)
+        if n_copy >= 2:
```

Comment:
It could have been done
```python
if inst.pointers.get('NCOPY', 1):
    inst._label_alternatives()
```

but this way gives me higher coverage :).